src/lv_fonts/lv_font_template.c

```diff
@@ -0,0 +1,172 @@
+
+#include "lvgl/lvgl.h"
+
+/***********************************************************************************
+ * Copy/Paste the command line instruction
+ **********************************************************************************/
+
+/*-----------------
+ *    BITMAPS
+ *----------------*/
+
+/*Store the image of the glyphs*/
+static const uint8_t gylph_bitmap[] = {
+    /*Unicode: U+0031 (1) */
+    0x38, 0xf8, 0x98, 0x18, 0x18, 0x18, 0x18, 0x18,
+    0x18, 0x18, 0x18, 0xff, 0xff,
+
+    /*Unicode: U+0033 (3) */
+    0x7e, 0x00, 0xff, 0x00, 0x83, 0x80, 0x01, 0x80,
+    0x03, 0x80, 0x3e, 0x00, 0x3f, 0x00, 0x03, 0x80,
+    0x01, 0x80, 0x01, 0x80, 0x83, 0x80, 0xff, 0x00,
+    0x7e, 0x00,
+
+    /*Unicode: U+0041 (A) */
+    0x06, 0x00, 0x06, 0x00, 0x0f, 0x00, 0x0f, 0x00,
+    0x19, 0x80, 0x19, 0x80, 0x19, 0x80, 0x30, 0xc0,
+    0x3f, 0xc0, 0x7f, 0xe0, 0x60, 0x60,  0x60, 0x60,
+    0xc0, 0x30,
+
+    /*Unicode: U+0061 (a) */
+    0x3c, 0x7e, 0x47, 0x03, 0x3f, 0xff, 0xc3, 0xc7,
+    0xff, 0x7b,
+};
+
+
+/*---------------------
+ *  GLYPH DESCRIPTION
+ *--------------------*/
+
+static lv_font_glyph_dsc_fmt_txt_t glyph_dsc[] = {
+        {.bitmap_index = 0,     .adv_w = 23,    .box_h = 13, .box_w = 9, .ofs_x = 2, .ofs_y = 5},
+        {.bitmap_index = 32,    .adv_w = 44,    .box_h = 12, .box_w = 7, .ofs_x = 1, .ofs_y = 3},
+        {.bitmap_index = 55,    .adv_w = 29,    .box_h = 11, .box_w = 8, .ofs_x = 0, .ofs_y = 4},
+};
+
+/*---------------------
+ *  CHARACTER MAPPING
+ *--------------------*/
+
+static uint16_t unicode_list_1 = {0x234, 0x295, 0x456};
+static uint16_t glyph_id_ofs_list_1 = {232, 545, 1466};
+
+static uint8_t glyph_id_ofs_list_3 = {32, 45, 66};
+
+
+/*Collect the unicode lists and glyph_id offsets*/
+static const lv_font_cmap_fmt_txt_t cmaps[] = {
+        {
+            .range_start = 0x280,  .range_length = 124,   .type = LV_FONT_FMT_TXT_CMAP_SPARSE_FULL,
+            .glyph_id_start = 223, .unicode_list = unicode_list_1,  .glyph_id_ofs_list = glyph_id_ofs_list_1
+        },
+        {
+            .range_start = 0x20,  .range_length = 112,     .type = LV_FONT_FMT_TXT_CMAP_FORMAT0_TINY,
+            .glyph_id_start = 456, .unicode_list = NULL,   .glyph_id_ofs_list = NULL
+        },
+        {
+            .range_start = 0x560, .range_length = 7654,    .type = LV_FONT_FMT_TXT_CMAP_FORMAT0_FULL,
+            .glyph_id_start = 756, .unicode_list = NULL,   .glyph_id_ofs_list = glyph_id_ofs_list_3},
+
+};
+
+/*-----------------
+ *    KERNING
+ *----------------*/
+
+/*Left glyphs for kering*/
+static uint16_t kern_left_gylph_ids[] =
+{
```

Comment:
That will not work with this kerning format (touples list). Pairs should be packed into `uint32_t` to allow fast binary search. If you split to left/right arrays - binsearch will be much more difficult.

Reply:
True.
Updated here (along with a few other things): 0acabe4

Reply:
Seems ok. I'm doing convertor right now. Only cmap/kern left. Should be ready today or tomorrow.

Reply:
Great!